packages/frontend/src/features/post/components/Post/Post.tsx

```diff
@@ -7,13 +7,7 @@ import Comment from '@/assets/img/comment.png'
 import Downvote from '@/assets/img/downvote.png'
 import Upvote from '@/assets/img/upvote.png'
 import { useAuthStatus } from '@/features/auth'
-import {
-    LikeAnimation,
-    VoteFailureDialog,
-    useVotes,
-    usePostStore,
```

Comment:
Do we need `usePostStore` anymore?

 `usePostStore` is not a dependency of other files currently.

Reply:
removed